### ISSUE TITLE
Building master via Cocoapods fails

### ISSUE THREAD

**Issue by scott-ce** [ResearchKit/ResearchKit#610]
I tried updating to the latest master in order to get the updates for PR #602 (we were running into the same layout issue displaying a chart in a table cell before the data source can fetch HealthKit data). Unfortunately, I'm unable to compile RK master because of a header issue: ORKWaitStepView.h fails to import ORKVerticalContainerView.h, because the latter isn't exposed as a public header.

In master, ORKWaitStepView.h appears in ResearchKit-umbrella.h. In release 1.3.0, it does not.

It appears this issue was introduced in commit 8c86b8d8, which updated the attributes of ORKWaitStepView.h in project.pbxproj and made it private. If I understand correctly, find_headers.rb determines the headers for the podspec by including the public & private headers. Now that ORKWaitStepView.h is private, it is included in the umbrella header, but ORKVerticalContainerView.h is not — so our build fails.

# Versions

Xcode 7.2 (7C68) building against commit 8c86b8d8 using Cocoapods 0.39.0.

# Steps to Reproduce

1. Create an application with a Podfile containing this dependency (54927408 is the latest master commit as I write this):
```
pod 'ResearchKit', :git => 'https://github.com/ResearchKit/ResearchKit.git', :commit => '54927408'
```
2. Run `pod install`.
3. Attempt to build the application.

# Expected Results

App builds successfully.

# Actual Results

```
<module-includes>:1:9: note: in file included from <module-includes>:1:
#import "/Users/USER/Library/Developer/Xcode/DerivedData/APPNAME-ewpkpsoawrrntgfcyxsmjufowntg/Build/Products/Debug-iphonesimulator/ResearchKit.framework/Headers/ResearchKit-umbrella.h"
        ^
/Users/USER/Library/Developer/Xcode/DerivedData/APPNAME-ewpkpsoawrrntgfcyxsmjufowntg/Build/Products/Debug-iphonesimulator/ResearchKit.framework/Headers/ResearchKit-umbrella.h:56:9: note: in file included from /Users/USER/Library/Developer/Xcode/DerivedData/APPNAME-ewpkpsoawrrntgfcyxsmjufowntg/Build/Products/Debug-iphonesimulator/ResearchKit.framework/Headers/ResearchKit-umbrella.h:56:
#import "ORKWaitStepView.h"
        ^
/Users/USER/Library/Developer/Xcode/DerivedData/APPNAME-ewpkpsoawrrntgfcyxsmjufowntg/Build/Products/Debug-iphonesimulator/ResearchKit.framework/Headers/ORKWaitStepView.h:41:9: error: 'ORKVerticalContainerView.h' file not found
#import "ORKVerticalContainerView.h"
        ^
/Users/USER/Development/Repos/APPNAME/APPNAME/Questionnaires/RKJSONDefinitions/RKJSONEnums.swift:10:8: error: could not build Objective-C module 'ResearchKit'
import ResearchKit
       ^
```

**Comment by scott-ce** [comment]
FYI, I updated the Steps to Reproduce section to reference the latest commit on master. The original report had a copy/paste error that might have made it more difficult for you to reproduce.

**Comment by scott-ce** [comment]
Believe it's okay to close this. Resolved by PR #612
